text_generation/causal_lm/cpp/continuous_batching/library/CMakeLists.txt

```diff
@@ -17,6 +17,18 @@ FetchContent_MakeAvailable(nlohmann_json)
 
 find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 
+function(ov_genai_build_jinja2cpp)
+    FetchContent_Declare(jinja2cpp
+	    jinja2cpp
+    GIT_REPOSITORY https://github.com/jinja2cpp/Jinja2Cpp.git
+    GIT_TAG 9ae7e1fc45d707e1686dd425a154d30963801944
+    GIT_SUBMODULES_RECURSE ON)
+    FetchContent_GetProperties(jinja2cpp)
+    FetchContent_MakeAvailable(jinja2cpp)
```

Comment:
we also need to add these lines to root cmake https://github.com/pavel-esir/openvino.genai/blob/generate_pipeline/CMakeLists.txt#L7-L18

otherwise, JinjaCpp will be built as ReleaseWithDebInfo and its size is huge.